Lib/test/test_unparse.py

```diff
@@ -321,6 +331,23 @@ def test_simple_expressions_parens(self):
         self.check_src_roundtrip("call((yield x))")
         self.check_src_roundtrip("return x + (yield x)")
 
+    def test_docstrings(self):
+        docstrings = (
+            '"""simple doc string"""',
+            '''"""A more complex one
+            with some newlines"""''',
+            '''"""Foo bar baz
+
+            empty newline"""''',
+            '"""With some \t"""',
+            '"""Foo "bar" baz """',
+        )
+
+        keywords = ("class", "def", "async def")
```

Comment:
We are missing a module object for the test.

I also think we need some negative cases, like functions where the string is assigned to and other similar situations.